dentapp/src/components/Header.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from "react";
+import { LinkContainer } from "react-router-bootstrap";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+
+import Navbar from "react-bootstrap/Navbar";
+import Button from "react-bootstrap/Button";
+import Container from "react-bootstrap/Container";
+
+function createBackButton(color) {
+  return (
+    <svg
+      width="9"
+      height="16"
+      viewBox="0 0 9 16"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <path
+        d="M8.7525 1.90245L7.425 0.574951L0 7.99995L7.425 15.425L8.7525 14.0975L2.655 7.99995L8.7525 1.90245Z"
+        fill={color}
+      />
+    </svg>
+  );
+}
+
+const HeadStyled = styled.div.attrs((props) => ({
+  backgroundColor: props.backgroundColor,
+  iconAndTitleColor: props.iconAndTitleColor,
+}))`
+  background-color: ${(props) => props.backgroundColor};
+  color: ${(props) => props.iconAndTitleColor} !important;
+  .Navbar {
+    flex-wrap: wrap !important;
+  }
+`;

```

Comment:
I think it would be better to don't mix the styled components and classes

**Check this example:**
const Button = styled.button`
  /* Adapt the colors based on primary prop */
  background: ${props => props.primary ? "palevioletred" : "white"};
  color: ${props => props.primary ? "white" : "palevioletred"};

  font-size: 1em;
  margin: 1em;
  padding: 0.25em 1em;
  border: 2px solid palevioletred;
  border-radius: 3px;
`;